core/lib/spree/core/engine.rb

```diff
@@ -116,12 +116,12 @@ class Engine < ::Rails::Engine
       end
 
       config.to_prepare do
-        # Load spree locales before decorators
+        # Ensure spree locale paths are present before decorators
         I18n.load_path += Dir.glob(
```

Comment:
Maybe just use `I18n.load_path = Dir.glob(` so it won't be added from the beginning to load_path? 
What do you think about such solution? Would it do the same or am I wrong and cannot see something here?

Reply:
@bbonislawski 
This is definitely not a way to go since we'll lose previously loaded translations.

This issue happens due to the fact that we're not ensuring that Spree locales are loaded first as opposed to being loaded last.

We could change the implementation to something like `I18n.load_path.unshift *Dir.glob(...)` which will fix the issue and preserve current behaviour.